cmd/p2p/sensor/sensor.go

```diff
@@ -326,6 +352,8 @@ increase CPU and memory usage.`)
 significantly increase CPU and memory usage.`)
 	SensorCmd.Flags().BoolVar(&inputSensorParams.ShouldRunPprof, "pprof", false, "Whether to run pprof")
 	SensorCmd.Flags().UintVar(&inputSensorParams.PprofPort, "pprof-port", 6060, "Port pprof runs on")
+	SensorCmd.Flags().BoolVar(&inputSensorParams.ShouldRunPrometheus, "prom", true, "Whether to run Prometheus")
+	SensorCmd.Flags().UintVar(&inputSensorParams.PrometheusPort, "prom-port", 2112, "Port Prometheus runs on")
```

Comment:
Why use this specific port? The default Prometheus port is 9090 I think.

Reply:
Funnily, this was actually the port I saw in the prometheus docs, wierd that they use something different there.